### PR TITLE
test: add bdd test for trutbloc vdr

### PR DESCRIPTION
**Pull request by fqutishat**
Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

File: test/bdd/vdr/trustbloc/pkg/vdr/vdr_steps.go [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+	return &Steps{bddContext: ctx, httpClient: &http.Client{}}
+}
+
+// RegisterSteps registers agent steps
+func (e *Steps) RegisterSteps(s *godog.Suite) {
+	s.Step(`^TrustBloc DID is created through "([^"]*)" with key type "([^"]*)" with signature suite "([^"]*)"$`,
+		e.createDIDBloc)
+	s.Step(`^Resolve created DID and validate key type "([^"]*)", signature suite "([^"]*)"$`,
+		e.resolveCreatedDID)
+	s.Step(`^Bloc VDRI is initialized with resolver URL "([^"]*)"$`, e.initBlocVDRIWithResolverURL)
```
Comment (troyronda) [inline test/bdd/vdr/trustbloc/pkg/vdr/vdr_steps.go:58]: TrustBloc VDR

File: test/bdd/vdr/trustbloc/features/did_method_e2e.feature [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+# SPDX-License-Identifier: Apache-2.0
+#
+
+@all
+@did_method_rest
+Feature: Using DID method REST API
+
+  @e2e_sidetree
+  Scenario Outline: create trustbloc did and resolve through sidetree-mock
+    Then Bloc VDRI is initialized with resolver URL "https://localhost:48326/sidetree/v1/identifiers"
```
Comment (troyronda) [inline test/bdd/vdr/trustbloc/features/did_method_e2e.feature:13]: TrustBloc VDR

File: test/bdd/vdr/trustbloc/pkg/vdr/vdr_steps.go (hunk 1 above)
Comment (fqutishat) [reply test/bdd/vdr/trustbloc/pkg/vdr/vdr_steps.go]: done

File: test/bdd/vdr/trustbloc/features/did_method_e2e.feature (hunk 2 above)
Comment (fqutishat) [reply test/bdd/vdr/trustbloc/features/did_method_e2e.feature]: done

**Review by bstasyszyn — APPROVED** [review]

**Review by sandrask — APPROVED** [review]

**Review by troyronda — APPROVED** [review]

**Pull request merged**